src/components/expression/expression-loader.js

```diff
@@ -211,18 +212,26 @@ genotet.ExpressionLoader.prototype.update = function(method, fileName,
  * @param {!Array<string>} originalNames Original names from previous zoom
  *      status.
  * @param {!Array<string>} removeNames Names need to be removed.
+ * @return {!Array<string>} result Revised names.
```

Comment:
Unclear description of return values. Why removing names would return a list? You need to explain in detail in comments.